anesthetic/plotting/_matplotlib/hist.py

```diff
@@ -28,6 +28,8 @@
 class HistPlot(_WeightedMPLPlot, _HistPlot):
     # noqa: disable=D101
     def _calculate_bins(self, data):
+        if self.logx:
+            data = np.log10(data)
```

Comment:
ideally we'd make sure these lines were covered by a test (to ensure they're doing what we think they're doing).

Reply:
This does however need covering in order to pass the checks!

Reply:
This is work in progress, #330 required slight re-writes which I am not done with.

Reply:
Coverage back up to 100%.